membership/views.py

```diff
@@ -1179,32 +1178,71 @@ def member_reg_form(request, title, pk):
             # user is not allowed to edit this member
             return redirect('dashboard')
 
+
     elif request.method == "POST":
         # admin/owner can add any new/existing user
         # admin/owner can edit any existing owner
         # user can edit only themselves
-
         form = MemberForm(request.POST)
         if form.is_valid():
-            user = get_or_create_user(form)
             if pk == 0 and request.user == membership_package.owner or request.user in membership_package.admins.all():
+                userExists = User.objects.filter(email=form.cleaned_data['email']).exists()
+                if userExists == False:
```

Comment:
this is camelCase, this is not used in python.
also should be
if not user_exists:
not
if user_exists == False:

Reply:
What if the user does exists? should return httpresponse with fail message.